File: Python/PRAK402-2510817210005-Daffa-Dziibaan-Khoiri.py
angka = int(input(""))

for i in range(1, angka, 1):
    if(i % 2 == 1):
        print(i, end =" ")

print("")
for i in range(angka, 1, -1):
    if(i % 2 == 0):
        print(i, end = " ")

File: C/PRAK402-2510817210005-Daffa-Dziibaan-Khoiri.c
#include <stdio.h>

int main(){

    int angka;

    scanf("%d", &angka);

    for(int i = 1; i <= angka; i++ ){
        if(i % 2 == 1){
            printf("%d ", i); 
        }
    }

    printf("\n");
    for(int i = angka; i >= 2; i-- ){
        if(i % 2 == 0){
            printf("%d ", i); 
        }
    }

    return 0;
}

